desafio19.sql

```diff
@@ -0,0 +1 @@
+SELECT COUNT(*) AS orders_count FROM northwind.orders WHERE employee_id = 5 OR employee_id = 6 AND (shipper_id = 2);
```

Comment:
@renanpbraga interessante como escreveu o código, não havia pensado em usar 'OR' e 'AND'.
Outra maneira possível, inclusive como melhoria de desempenho é: 
```suggestion
SELECT COUNT(employee_id) AS orders_count FROM northwind.orders WHERE orders.shipper_id = 2 AND orders.employee_id IN (5,6);
```